src/Relation/Binary/Properties/ApartnessRelation.agda

```diff
@@ -0,0 +1,33 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Apartness properties
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Relation.Binary.Core using (Rel)
+
+module Relation.Binary.Properties.ApartnessRelation
+  {a ℓ₁ ℓ₂} {A : Set a}
+  {_≈_ : Rel A ℓ₁}
+  {_#_ : Rel A ℓ₂}
+  where
+
+open import Relation.Binary.Definitions using (Reflexive)
+open import Relation.Binary.Consequences using (comp⇒¬-trans)
+open import Relation.Binary.Structures using (IsEquivalence; IsApartnessRelation)
+open import Relation.Nullary using (¬_)
+
+
+private
+  _¬#_ : A → A → Set _
+  x ¬# y = ¬ (x # y)
+
+¬#-equiv : Reflexive _≈_ → IsApartnessRelation _≈_ _#_ → IsEquivalence _¬#_
+¬#-equiv re apart = record
+  { refl = irrefl re
+  ; sym = λ x¬#y y#x → x¬#y (sym y#x)
```

Comment:
Is this `sym⇒¬-sym`?

Reply:
Yes, and I've replaced it. One odd thing that maybe you could comment on: for some reason I have to give agda the implicit arguments to `sym`.

```agda
¬#-isEquivalence : Reflexive _≈_ → IsApartnessRelation _≈_ _#_ → IsEquivalence _¬#_
¬#-isEquivalence re apart = record
  { refl = irrefl re
  ; sym = λ {a} {b} → sym⇒¬-sym sym {a} {b} 
  ; trans = cotrans⇒¬-trans cotrans
  } where open IsApartnessRelation apart 
 ```

I'm not sure why...

Reply:
Yes, I also run into this problem now and again. I've never managed to find an explanation I can understand. We run into the problem in other places in the library as well, so it's fine to leave the implicits in.